frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -15,7 +15,7 @@ import * as DATABASE from '../../constants/database.js';
  *    containing the query results with zero or more Trip  documents.
  */
 function queryUserTrips(db, userEmail) {
-  return db.collection(DATABASE.COLLECTION_TRIPS)
+  return null.collection(DATABASE.COLLECTION_TRIPS)
```

Comment:
Why is this null?

Reply:
Whoops. This was from testing the capability of the the error handling in `componentDidMount()` in the case that the Firestore instance was null. This should be updated now.